### DISCUSSION TITLE
Welcome to StockTV Discussions!

### DISCUSSION THREAD

**Discussion by Trawacho** [Trawacho/StockTV#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by unclasp** [comment]
Ist es möglich das auf einem alten HP Pavillon DM 1 zu installieren? Der hatt im Moment ein altes Linux drauf. Lizenz nur für Windows 7 dabei. Windows IoT auf den HP zu installieren geht anscheinend nicht. Tolle Arbeit! LG

**Reply by Trawacho** [reply]
Hab ich selber noch nicht getestet. Aber sollte grundsätzlich klappen. Die App kann dann unter Windows im Kiosk Modus laufen. Die Installation kann  über das Powershell Skript erfolgen.

**Reply by unclasp** [reply]
Die Powershell Skript Intallation funktioniert auch unter Windows 7? Ich bekomms nicht hin

**Reply by Trawacho** [reply]
Wie gesagt. Ich hab's noch nie probiert. Getestet bisher nur unter Windows 10 mit Raspi4 (Stichwort: WindowsOnRaspi) oder mit "normalem" Windows10 auf beliebige Rechner.
Bin momentan etwas verhindert um besser helfen zu können....aber was klappt nicht?

**Reply by unclasp** [reply]
Bekomme bei dem Versuch es zu installieren diese Fehlermeldungen: https://i.imgur.com/JWyhmxv.jpg

**Reply by Trawacho** [reply]
Hab mir es nochmal angeschaut. Die Minimalversion ist "Windows 10, version 1803 (10.0); Build 17134)". Somit wird es mit Windows7 nicht funktionieren.

**Reply by unclasp** [reply]
Alles gut, ist auch so mit dem alten 2B+ möglich. Alles nur weil die Raspberry's so teuer geworden sind. Die PC App stürtzt ab wenn ich die Ergebnisse holen will, aber das klären wir nächstes mal. Toll wie du das gemacht hast, finde die Anwendung einfach gut durchdacht! Danke nochmal für deine Bemühungen! Danke für den Support! LG Markus

**Comment by unclasp** [comment]
Hallo,
ist es möglich die App in Windows 11 irgendwie zu den Autostart Apps zu bekommen? Finde keinen Speicherpfad und kann keine Verknüpfung erstellen.

Danke
LG

**Reply by Trawacho** [reply]
Hallo,
suche doch mal nach Windows11 Kiosk Mode.
z.B.: hier(https://learn.microsoft.com/de-de/windows/configuration/kiosk-single-app)

**Reply by unclasp** [reply]
Danke, hat geklappt!

**Comment by unclasp** [comment]
Hallo,
alles funktioniert super! Die Spielerkollegen haben eine Frage: Ist es möglich die Farben zu tauschen (rot-grün). Rot sollte links sein und grün auf der rechten Seite.
Vielen Dank für die Bemühungen!
LG Markus

**Reply by bachmichl** [reply]
Nur GRÜN hat eine feste Position auf der Tastatur. Die Tasten links und rechts neben der grünen Taste haben beide die Funktion ROT

![IMG-20210621-WA0000](https://user-images.githubusercontent.com/94722656/233060641-d1d9bd30-e4cc-49c8-b0c5-05372b61b320.jpg)
![IMG-20210621-WA0001](https://user-images.githubusercontent.com/94722656/233060656-3bcf0a17-17b6-4d32-9912-163451ed49e9.jpg)

Also einfach tauschen. Es hat sich als sinnvoll erwiesen, je nach Montageposition, die rote Taste an der Position der roten Anzeige zu orientieren (unterschiedlich an den Häusern)

**Reply by unclasp** [reply]
Danke für die schnelle Antwort, ich meinte aber die Anzeige am Monitor. Dort hätten wir gerne die Farben vertauscht.

**Reply by bachmichl** [reply]
ich dachte, dass dies am Regelbuch liegt. Die Farbe ändert sich auch automatisch, wenn man im Setup 'nächste Bahn' zwischen links und rechts ändert. Ungerade Startnummern sind 'grün'. Dies bedeutet, wenn 'nächste Bahn' auf rechts gesetzt ist, muss die Anzeige auf Bahn 1 links grün sein (dort startet Nr. 1) -  wenn ich es jetzt nicht verwechselt habe

Auszug Regelbuch:
Stockmarker mit der Bezeichnung „IFI-gerecht“ in den Farben RAL 1026 (leuchtgelb) und RAL 3024 (leuchtrot) sind im Mannschaftsspiel auf Anweisung des Wettbewerbsleiters zu verwenden; leuchtgelb für die ungeraden Startnummern und leuchtrot für die geraden. Ein 
Wechsel der Stockmarker erfolgt für die Mannschaft, 
die in die Pause geht mit der, die aus der Pause kommt.

**Reply by Trawacho** [reply]
Wie bachmichl schreibt, durch die Umstellung in den Settings kann man die Farben tauschen. Zu beachten ist, Mannschaft 1 hat beim ersten Spiel auf Bahn 1 die Farbe GRÜN.

**Reply by unclasp** [reply]
Super Danke, hat funktioniert! AllesTOP! Danke für den Support!
LG
Markus

**Comment by BernieCe71** [comment]
Servus!
Ist es möglich im Best off Modus die eingegebenen Ergebnisse nach Spielende mit einer Tastenkombi zu löschen oder muss man mit der blauen Taste jede einzelne Kehre löschen?

**Comment by bachmichl** [comment]
Servus,
 
ich weiss nicht, ob schon geantwortet wurde. Bei mir landete die Nachricht im Spam.
 
Best Of kann ich leider nicht beantworten. Zum Löschen ist zu beachten, dass eine '0' in der Mitte steht, dann ist die Entprellung der blauen Taste inaktiv. D.h. '0' eingeben, jedoch ohne rot oder grün zu tippen - damit steht die '0' unten in der Mitte. Dann ist das Löschen sehr schnell.
 
Servus,
Mich


Gesendet: Montag, 4. August 2025 um 15:16
Von: BernieCe71 ***@***.***>
An: Trawacho/StockTV ***@***.***>
CC: bachmichl ***@***.***>,Comment ***@***.***>
Betreff: Re: [Trawacho/StockTV] Welcome to StockTV Discussions! (Discussion #5)

 
Servus!Ist es möglich im Best off Modus die eingegebenen Ergebnisse nach Spielende mit einer Tastenkombi zu löschen oder muss man mit der blauen Taste jede einzelne Kehre löschen?

**Reply by BernieCe71** [reply]
Danke Mich, 
für deine Antwort, werde ich demnächst ausprobieren.
VG Berni
